demo/app-root.ts

```diff
@@ -1,20 +1,36 @@
-import { html, css, LitElement } from 'lit';
+import { css, html, LitElement } from 'lit';
 import { customElement } from 'lit/decorators.js';
-import '../src/your-webcomponent';
+
+import type { Review } from '@internetarchive/metadata-service';
+import '../src/review-form';
+
+const mockOldReview: Review = {
```

Comment:
To create a Review object, I would instantiate it with a JSON blob since that's closer to how it's going to come from the API.. ie:

```
const mockOldReview = new Review({
  stars: '3',
  review title: 'What a cool book!',
  ...
  reviewdate: '2025-03-03 18:13:36',
  createdate: '2025-02-25 14:28:19'
})
```

The Review object will handle the parsing of the dates and `stars` values.